__tests__/api.test.js

```diff
@@ -0,0 +1,43 @@
+const app = require('../app')
+const request = require('supertest')
+const db = require('../db/connection')
+const seed = require('../db/seeds/seed')
+const testData = require('../db/data/test-data')
+
+
+beforeEach(() => seed(testData))
+
+afterAll(() => db.end())
+
+describe("News API BACKEND PROJECT", () => {
+    describe("GET /api/topics that returns all topics", () => {
+
+        test('returns a 200 response status', () => {
+            return request(app)
+            .get('/api/topics')
+            .expect(200)
+        })
+        test('returns a 200 response status and data with correct properties', () => {
+            return request(app)
+            .get('/api/topics')
+            .expect(200)
+            .then((response) => {
+                const { body } = response
+                expect(Array.isArray(body)).toBe(true)
+                body.forEach((topic) => {
+                    expect(topic).toHaveProperty("slug")
+                    expect(topic).toHaveProperty("description")
+                })
+            })
+        })
+        test('returns a 404 response status if sent invalid address with appropriate error message', () => {
+            return request(app)
+            .get('/api/topicks')
+            .expect(404)
+            .then((response) => {
+                const { body } = response
+                expect(body.msg).toBe("Sorry, the endpoint you are searching for does not exist.")
+            })
+        })
```

Comment:
Really nice error handling test, but this should not be in the describe for /api/topics as this logic does not rely on anything from that functionality. 

You should only have error handling in the same describe block if it is to do with the endpoint you are working on. This specific case is any invalid endpoint and therefore should belong in its own describe block. This test would still pass for /api/banana which is also just an invalid endpoint.